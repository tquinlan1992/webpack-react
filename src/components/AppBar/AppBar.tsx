import * as React from 'react';
import AppBarMaterialUi from 'material-ui/AppBar';
import './index.scss';
const qaLogo = require('./qa_logo.png');

class AppBar extends React.Component<any, any> {
    constructor(props: any) {
        super(props);

    }

    public render() {
        return (
            <div>
            <img src={qaLogo}/>
                <AppBarMaterialUi className='AppBar' title="Title" iconClassNameRight="muidocs-icon-navigation-expand-more">
                </AppBarMaterialUi>
            </div>
        )
    }
}

export default AppBar;
