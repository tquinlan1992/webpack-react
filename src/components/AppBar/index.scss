@import "../../style/colors";

.AppBar {
    background-color: $tqGreen !important;
}
