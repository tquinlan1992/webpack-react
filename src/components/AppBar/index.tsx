import * as React from 'react';
import AppBarMaterialUi from 'material-ui/AppBar';
import {Tabs, Tab} from 'material-ui/Tabs';
const styles = {
  appBar: {
    flexWrap: 'wrap',
  },
  tabs: {
    width: '100%',
  },
};
const qaLogo = require('./qa_logo.png');

class AppBar extends React.Component<any, any> {
    constructor(props: any) {
        super(props);

    }

    public render() {
        return (
            <div>
                <AppBarMaterialUi className='AppBar' title="Title" iconClassNameRight="muidocs-icon-navigation-expand-more">
                <Tabs style={styles.tabs}>
                <img src={qaLogo} />
                  <Tab label='first tab'>First tab in docker text</Tab>
                  <Tab label='second tab'>Second tab text</Tab>
                  <Tab label='third tab'>Third tab text</Tab>
                </Tabs>
                </AppBarMaterialUi>
            </div>
        )
    }
}

export default AppBar;
