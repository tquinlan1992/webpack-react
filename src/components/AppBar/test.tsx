// Link.react-test.js
import * as React from 'react';
import AppBar from './index';
import * as renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

test('see app bar', () => {
    let header = "header is working";

    const component = renderer.create(
        <MuiThemeProvider>
            <AppBar title="Title2" iconClassNameRight="muidocs-icon-navigation-expand-more" header={header} headerChangeText="header changed">Facebook</AppBar>
        </MuiThemeProvider>
    );
    debugger;

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    component.toJSON().children[0].props.onClick();

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
