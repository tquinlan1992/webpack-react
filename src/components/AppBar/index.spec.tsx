// Link.react-test.js
import * as React from 'react';
import AppBar from './index';
import * as renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import {
    BrowserRouter
} from 'react-router-dom';

test('see app bar', () => {
    let header = "header is working";

    const component = renderer.create(
        <BrowserRouter>
        <MuiThemeProvider>
            <AppBar title="Title2" iconClassNameRight="muidocs-icon-navigation-expand-more" header={header} headerChangeText="header changed">Facebook</AppBar>
        </MuiThemeProvider>
        </BrowserRouter>
    );
    debugger;

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    component.toJSON().children[0].props.onClick();

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
