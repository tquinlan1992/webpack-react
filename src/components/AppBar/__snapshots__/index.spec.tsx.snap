// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`see app bar 1`] = `
<div
  className="AppBar"
  style={
    Object {
      "WebkitTapHighlightColor": "rgba(0,0,0,0)",
      "backgroundColor": "#00bcd4",
      "borderRadius": "0px",
      "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
      "boxSizing": "border-box",
      "color": "rgba(0, 0, 0, 0.87)",
      "display": "flex",
      "fontFamily": "Roboto, sans-serif",
      "muiPrepared": true,
      "paddingLeft": 24,
      "paddingRight": 24,
      "position": "relative",
      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
      "width": "100%",
      "zIndex": 1100,
    }
  }
>
  <button
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseOut={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
        "background": "none",
        "border": 10,
        "boxSizing": "border-box",
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": 0,
        "fontWeight": "inherit",
        "height": 48,
        "margin": 0,
        "marginLeft": -16,
        "marginRight": 8,
        "marginTop": 8,
        "muiPrepared": true,
        "outline": "none",
        "overflow": "visible",
        "padding": 12,
        "position": "relative",
        "textDecoration": "none",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "verticalAlign": null,
        "width": 48,
      }
    }
    tabIndex={0}
    type="button"
  >
    <div
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
    >
      <svg
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "color": "#ffffff",
            "display": "inline-block",
            "fill": "#ffffff",
            "height": 24,
            "muiPrepared": true,
            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "userSelect": "none",
            "width": 24,
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </div>
  </button>
  <h1
    onClick={[Function]}
    style={
      Object {
        "boxFlex": 1,
        "color": "#ffffff",
        "flex": "1",
        "fontSize": 24,
        "fontWeight": 400,
        "height": 64,
        "letterSpacing": 0,
        "lineHeight": "64px",
        "margin": 0,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingTop": 0,
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
      }
    }
  >
    Title
  </h1>
  <button
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseOut={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
        "background": "none",
        "border": 10,
        "boxSizing": "border-box",
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": 0,
        "fontWeight": "inherit",
        "height": 48,
        "margin": 0,
        "marginLeft": "auto",
        "marginRight": -16,
        "marginTop": 8,
        "muiPrepared": true,
        "outline": "none",
        "overflow": "visible",
        "padding": 12,
        "position": "relative",
        "textDecoration": "none",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "verticalAlign": null,
        "width": 48,
      }
    }
    tabIndex={0}
    type="button"
  >
    <div
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
    >
      <span
        className="muidocs-icon-navigation-expand-more"
        color="rgba(0, 0, 0, 0.87)"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "color": "#ffffff",
            "display": "inline-block",
            "fill": "#ffffff",
            "fontSize": 24,
            "muiPrepared": true,
            "position": "relative",
            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "userSelect": "none",
          }
        }
      />
    </div>
  </button>
  <div
    style={
      Object {
        "muiPrepared": true,
        "width": "100%",
      }
    }
    value="/first"
  >
    <div
      style={
        Object {
          "backgroundColor": "#00bcd4",
          "display": "flex",
          "muiPrepared": true,
          "whiteSpace": "nowrap",
          "width": "100%",
        }
      }
    >
      <button
        disabled={undefined}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        style={
          Object {
            "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
            "background": "none",
            "border": 10,
            "boxSizing": "border-box",
            "color": "#ffffff",
            "cursor": "pointer",
            "display": "inline-block",
            "fontFamily": "Roboto, sans-serif",
            "fontSize": 14,
            "fontWeight": 500,
            "margin": 0,
            "muiPrepared": true,
            "outline": "none",
            "padding": 0,
            "position": "relative",
            "textDecoration": "none",
            "textTransform": "uppercase",
            "verticalAlign": null,
            "width": "33.333333333333336%",
          }
        }
        tabIndex={0}
        type="button"
      >
        <div
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
        >
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flexDirection": "column",
                "height": 48,
                "justifyContent": "center",
              }
            }
          >
            first tab
          </div>
        </div>
      </button>
      <button
        disabled={undefined}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        style={
          Object {
            "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
            "background": "none",
            "border": 10,
            "boxSizing": "border-box",
            "color": "rgba(255, 255, 255, 0.7)",
            "cursor": "pointer",
            "display": "inline-block",
            "fontFamily": "Roboto, sans-serif",
            "fontSize": 14,
            "fontWeight": 500,
            "margin": 0,
            "muiPrepared": true,
            "outline": "none",
            "padding": 0,
            "position": "relative",
            "textDecoration": "none",
            "textTransform": "uppercase",
            "verticalAlign": null,
            "width": "33.333333333333336%",
          }
        }
        tabIndex={0}
        type="button"
      >
        <div
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
        >
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flexDirection": "column",
                "height": 48,
                "justifyContent": "center",
              }
            }
          >
            second tab
          </div>
        </div>
      </button>
      <button
        disabled={undefined}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        style={
          Object {
            "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
            "background": "none",
            "border": 10,
            "boxSizing": "border-box",
            "color": "rgba(255, 255, 255, 0.7)",
            "cursor": "pointer",
            "display": "inline-block",
            "fontFamily": "Roboto, sans-serif",
            "fontSize": 14,
            "fontWeight": 500,
            "margin": 0,
            "muiPrepared": true,
            "outline": "none",
            "padding": 0,
            "position": "relative",
            "textDecoration": "none",
            "textTransform": "uppercase",
            "verticalAlign": null,
            "width": "33.333333333333336%",
          }
        }
        tabIndex={0}
        type="button"
      >
        <div
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
        >
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flexDirection": "column",
                "height": 48,
                "justifyContent": "center",
              }
            }
          >
            third tab
          </div>
        </div>
      </button>
    </div>
    <div
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <div
        style={
          Object {
            "backgroundColor": "#ff4081",
            "bottom": 0,
            "display": "block",
            "height": 2,
            "left": "0%",
            "marginTop": -2,
            "muiPrepared": true,
            "position": "relative",
            "transition": "left 1s cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "width": "33.333333333333336%",
          }
        }
      />
    </div>
    <div
      className={undefined}
      style={
        Object {
          "muiPrepared": true,
        }
      }
    >
      <div
        style={
          Object {
            "position": "relative",
            "textAlign": "initial",
            "width": "100%",
          }
        }
      >
        header is working
         First tab a in docker text
      </div>
      <div
        style={
          Object {
            "height": 0,
            "overflow": "hidden",
            "position": "relative",
            "textAlign": "initial",
            "width": "100%",
          }
        }
      >
        Second tab text 
        <img
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </div>
      <div
        style={
          Object {
            "height": 0,
            "overflow": "hidden",
            "position": "relative",
            "textAlign": "initial",
            "width": "100%",
          }
        }
      >
        Third tab text2
      </div>
    </div>
  </div>
</div>
`;

exports[`see app bar 2`] = `
<div
  className="AppBar"
  style={
    Object {
      "WebkitTapHighlightColor": "rgba(0,0,0,0)",
      "backgroundColor": "#00bcd4",
      "borderRadius": "0px",
      "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
      "boxSizing": "border-box",
      "color": "rgba(0, 0, 0, 0.87)",
      "display": "flex",
      "fontFamily": "Roboto, sans-serif",
      "muiPrepared": true,
      "paddingLeft": 24,
      "paddingRight": 24,
      "position": "relative",
      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
      "width": "100%",
      "zIndex": 1100,
    }
  }
>
  <button
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseOut={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
        "background": "none",
        "border": 10,
        "boxSizing": "border-box",
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": 0,
        "fontWeight": "inherit",
        "height": 48,
        "margin": 0,
        "marginLeft": -16,
        "marginRight": 8,
        "marginTop": 8,
        "muiPrepared": true,
        "outline": "none",
        "overflow": "visible",
        "padding": 12,
        "position": "relative",
        "textDecoration": "none",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "verticalAlign": null,
        "width": 48,
      }
    }
    tabIndex={0}
    type="button"
  >
    <div
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
    >
      <svg
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "color": "#ffffff",
            "display": "inline-block",
            "fill": "#ffffff",
            "height": 24,
            "muiPrepared": true,
            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "userSelect": "none",
            "width": 24,
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </div>
  </button>
  <h1
    onClick={[Function]}
    style={
      Object {
        "boxFlex": 1,
        "color": "#ffffff",
        "flex": "1",
        "fontSize": 24,
        "fontWeight": 400,
        "height": 64,
        "letterSpacing": 0,
        "lineHeight": "64px",
        "margin": 0,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingTop": 0,
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
      }
    }
  >
    Title
  </h1>
  <button
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseOut={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
        "background": "none",
        "border": 10,
        "boxSizing": "border-box",
        "cursor": "pointer",
        "display": "inline-block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": 0,
        "fontWeight": "inherit",
        "height": 48,
        "margin": 0,
        "marginLeft": "auto",
        "marginRight": -16,
        "marginTop": 8,
        "muiPrepared": true,
        "outline": "none",
        "overflow": "visible",
        "padding": 12,
        "position": "relative",
        "textDecoration": "none",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "verticalAlign": null,
        "width": 48,
      }
    }
    tabIndex={0}
    type="button"
  >
    <div
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
    >
      <span
        className="muidocs-icon-navigation-expand-more"
        color="rgba(0, 0, 0, 0.87)"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "color": "#ffffff",
            "display": "inline-block",
            "fill": "#ffffff",
            "fontSize": 24,
            "muiPrepared": true,
            "position": "relative",
            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "userSelect": "none",
          }
        }
      />
    </div>
  </button>
  <div
    style={
      Object {
        "muiPrepared": true,
        "width": "100%",
      }
    }
    value="/first"
  >
    <div
      style={
        Object {
          "backgroundColor": "#00bcd4",
          "display": "flex",
          "muiPrepared": true,
          "whiteSpace": "nowrap",
          "width": "100%",
        }
      }
    >
      <button
        disabled={undefined}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        style={
          Object {
            "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
            "background": "none",
            "border": 10,
            "boxSizing": "border-box",
            "color": "#ffffff",
            "cursor": "pointer",
            "display": "inline-block",
            "fontFamily": "Roboto, sans-serif",
            "fontSize": 14,
            "fontWeight": 500,
            "margin": 0,
            "muiPrepared": true,
            "outline": "none",
            "padding": 0,
            "position": "relative",
            "textDecoration": "none",
            "textTransform": "uppercase",
            "verticalAlign": null,
            "width": "33.333333333333336%",
          }
        }
        tabIndex={0}
        type="button"
      >
        <div
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
        >
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flexDirection": "column",
                "height": 48,
                "justifyContent": "center",
              }
            }
          >
            first tab
          </div>
        </div>
      </button>
      <button
        disabled={undefined}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        style={
          Object {
            "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
            "background": "none",
            "border": 10,
            "boxSizing": "border-box",
            "color": "rgba(255, 255, 255, 0.7)",
            "cursor": "pointer",
            "display": "inline-block",
            "fontFamily": "Roboto, sans-serif",
            "fontSize": 14,
            "fontWeight": 500,
            "margin": 0,
            "muiPrepared": true,
            "outline": "none",
            "padding": 0,
            "position": "relative",
            "textDecoration": "none",
            "textTransform": "uppercase",
            "verticalAlign": null,
            "width": "33.333333333333336%",
          }
        }
        tabIndex={0}
        type="button"
      >
        <div
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
        >
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flexDirection": "column",
                "height": 48,
                "justifyContent": "center",
              }
            }
          >
            second tab
          </div>
        </div>
      </button>
      <button
        disabled={undefined}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        style={
          Object {
            "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
            "background": "none",
            "border": 10,
            "boxSizing": "border-box",
            "color": "rgba(255, 255, 255, 0.7)",
            "cursor": "pointer",
            "display": "inline-block",
            "fontFamily": "Roboto, sans-serif",
            "fontSize": 14,
            "fontWeight": 500,
            "margin": 0,
            "muiPrepared": true,
            "outline": "none",
            "padding": 0,
            "position": "relative",
            "textDecoration": "none",
            "textTransform": "uppercase",
            "verticalAlign": null,
            "width": "33.333333333333336%",
          }
        }
        tabIndex={0}
        type="button"
      >
        <div
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
        >
          <div
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flexDirection": "column",
                "height": 48,
                "justifyContent": "center",
              }
            }
          >
            third tab
          </div>
        </div>
      </button>
    </div>
    <div
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <div
        style={
          Object {
            "backgroundColor": "#ff4081",
            "bottom": 0,
            "display": "block",
            "height": 2,
            "left": "0%",
            "marginTop": -2,
            "muiPrepared": true,
            "position": "relative",
            "transition": "left 1s cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "width": "33.333333333333336%",
          }
        }
      />
    </div>
    <div
      className={undefined}
      style={
        Object {
          "muiPrepared": true,
        }
      }
    >
      <div
        style={
          Object {
            "position": "relative",
            "textAlign": "initial",
            "width": "100%",
          }
        }
      >
        header changed
         First tab a in docker text
      </div>
      <div
        style={
          Object {
            "height": 0,
            "overflow": "hidden",
            "position": "relative",
            "textAlign": "initial",
            "width": "100%",
          }
        }
      >
        Second tab text 
        <img
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </div>
      <div
        style={
          Object {
            "height": 0,
            "overflow": "hidden",
            "position": "relative",
            "textAlign": "initial",
            "width": "100%",
          }
        }
      >
        Third tab text2
      </div>
    </div>
  </div>
</div>
`;
